tests/ctl/cli/test_cli.py

```diff
@@ -930,3 +929,17 @@ def test_user_permissions(
         )
         print(result.output)
         assert result.exit_code == 0
+
+    @pytest.mark.unit
+    def test_user_permissions_not_found(
+        self, test_config_path: str, test_cli_runner: CliRunner, credentials_path: str
+    ) -> None:
+        """Test getting user permissions but the credentials file doesn't exit."""
+        print(credentials_path)
+        result = test_cli_runner.invoke(
+            cli,
+            ["-f", test_config_path, "user", "permissions"],
+            env={"FIDES_CREDENTIALS_PATH": "/root/notarealfile.credentials"},
+        )
+        print(result.output)
```

Comment:
Do the prints still need to be here?
```suggestion

```

Reply:
yep! In tests, print statements are only shown if the test fails. Extremely helpful for debugging and only shows up when its needed 🙂